README.md

```diff
@@ -4,7 +4,8 @@
 This repository is intended for the reference Jenkins configuration as code as well as JobDSL library
 
 ### Getting started
-TBD
+#### Requirements
+* Create backup directories to store jobs history. Make sure that backup directories exist before start up and owned by user 1000. If you don't these directories, create them and do chown 1000 <> 
```

Comment:
It is far away from being done. Would you be happy yourself with such kind of documentation? If you would have to deploy this project in one year from now would you understand what directories to create? Where? How many? Give example commands.

Use this as an example of proper documentation https://docs.docker.com/engine/installation/linux/centos/.